pkg/webhook/validates/crd_validate.go

```diff
@@ -432,6 +424,29 @@ func (v *securityPolicyValidator) validateRule(rule securityv1alpha1.Rule) error
 	return nil
 }
 
+func (v *securityPolicyValidator) validateIPBlock(ipBlock networkingv1.IPBlock) error {
+	_, cidrIPNet, err := net.ParseCIDR(ipBlock.CIDR)
+	if err != nil {
+		return fmt.Errorf("unvalid cidr %s: %s", ipBlock.CIDR, err)
+	}
+
+	for _, exceptCIDR := range ipBlock.Except {
+		_, exceptIPNet, err := net.ParseCIDR(exceptCIDR)
+		if err != nil {
+			return fmt.Errorf("unvalid except cidr %s: %s", exceptCIDR, err)
+		}
+
+		cidrMaskLen, _ := cidrIPNet.Mask.Size()
+		exceptMaskLen, _ := exceptIPNet.Mask.Size()
+
+		if !cidrIPNet.Contains(exceptIPNet.IP) || cidrMaskLen >= exceptMaskLen {
```

Comment:
cidrMaskLen >= exceptMaskLen ?

Reply:
If `cidrMaskLen >= exceptMaskLen`, then the cidr would have fewer host numbers, cidr cannot contain contain except.
Follow https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/networking/validation/validation.go#L187~L214.